src/app/components/Copilot/Copilot.tsx

```diff
@@ -103,27 +77,20 @@ export function CopilotChat(): React.JSX.Element {
 
   // for setting the initial copilot chat that takes place on page load.
   useEffect(() => {
-    const getInitialChat = async () => {
-      let _name: string;
-
-      await new Pieces.ConversationsApi()
-        .conversationsSnapshot({})
-        .then((output) => {
-          if (
-            output.iterable.length > 0 &&
-            output.iterable.at(0).hasOwnProperty("name")
-          ) {
-            _name = output.iterable.at(0).name;
-            GlobalConversationID = output.iterable.at(0).id;
+      const getInitialChat = async () => {
+          try {
+              const allConversations = await piecesClient.getConversations();
```

Comment:
Hey @VishalPawar1010 great work with this implementation. I was just wondering if you would like to also implement an additional functionality actually to list all of the conversations using the getConversations endpoint. on the UI. right now the back and forward buttons are not functionality. We can have a conversations list. and let the user select which conversation they wish to view. And you can use the piecesClient to then return that conversation ID and display the existing conversation in the Copilot chat window 
<img width="1338" alt="image" src="https://github.com/user-attachments/assets/be6f4936-70a0-4b85-a563-8c1d6aacd982">

Reply:
So we can remove the back and forward buttons and have a UI button for: choose copilot conversation

Reply:
What are your thoughts @Arindam-at-Pieces

Reply:
Sure, I will implement this functinality to list conversations and give option to select then display selected conversation.
Do we have any plan to use TailwindCss in future for UI feature additions (like dropdown etc) like this ?

Reply:
Yes I think we can definitely introduce Tailwind @VishalPawar1010 
I think that is a larger discussion as part to restructure the UI / Revamp the entire project, so that can definitely be explored.

Reply:
Ok then, until it is finalised I will implement UI in Vanila Js + Css.

Reply:
@shivay-at-pieces Added the mentioned functionality

Test video in Mac: 
https://github.com/user-attachments/assets/a01bc9df-3c57-4bff-ae7a-abfe75b024a7

Reply:
Hey @VishalPawar1010 what I will recommend is if you can also display the messages of the specific chat that you have selected in the chat window

Reply:
Okay sure. Will try to display message as well.

Reply:
Done.
Test Video : https://www.awesomescreenshot.com/video/29878767?key=e661acd7694e4055f75144ae76d61c5d